src/widget/sidebar_filler.c

```diff
@@ -0,0 +1,289 @@
+#include "sidebar_filler.h"
+
+#include "building/menu.h"
+#include "city/labor.h"
+#include "city/message.h"
+#include "city/population.h"
+#include "city/ratings.h"
+#include "city/view.h"
+#include "city/warning.h"
+#include "core/config.h"
+#include "core/direction.h"
+#include "core/lang.h"
+#include "core/string.h"
+#include "game/orientation.h"
+#include "game/settings.h"
+#include "game/state.h"
+#include "game/undo.h"
+#include "graphics/arrow_button.h"
+#include "graphics/graphics.h"
+#include "graphics/image.h"
+#include "graphics/image_button.h"
+#include "graphics/lang_text.h"
+#include "graphics/menu.h"
+#include "graphics/panel.h"
+#include "graphics/screen.h"
+#include "graphics/text.h"
+#include "graphics/window.h"
+#include "map/orientation.h"
+#include "scenario/property.h"
+#include "scenario/criteria.h"
+#include "sound/effect.h"
+#include "widget/city.h"
+#include "widget/minimap.h"
+#include "widget/sidebar.h"
+#include "window/advisors.h"
+#include "window/build_menu.h"
+#include "window/city.h"
+#include "window/empire.h"
+#include "window/message_dialog.h"
+#include "window/message_list.h"
+#include "window/mission_briefing.h"
+#include "window/overlay_menu.h"
+
+#define EXTRA_INFO_LINE_SPACE 32
+#define EXTRA_INFO_HEIGHT_GAME_SPEED 64
+#define EXTRA_INFO_HEIGHT_UNEMPLOYMENT 112
+#define EXTRA_INFO_HEIGHT_RATINGS 272
+#define EXTRA_INFO_TOP_PADDING 10;
+#define EXTRA_INFO_SPEED_PADDING 26;
+#define EXTRA_INFO_UNEMPLOYMENT_TOP_PADDING 20;
+#define EXTRA_INFO_UNEMPLOYMENT_BOTTOM_PADDING 22;
+
+typedef struct {
+    int value;
+    int target;
+} objective;
+
+typedef struct {
+    int height;
+    int game_speed;
+    int unemployment_percentage;
+    int unemployment_amount;
+    objective culture;
+    objective prosperity;
+    objective peace;
+    objective favor;
+    objective population;
+} extra_info;
+
+extra_info extra_info_vals;
```

Comment:
Make it static to prevent polluting the global scope and call it `data` (project convention).
For such file-specific data structs: don't define a typedef for the struct, but do the definition in one go:
```
static struct {
   ...
} data;
```
Also, put it below the function prototypes and arrow buttons.